velox/functions/lib/aggregates/tests/utils/AggregationTestBase.cpp

```diff
@@ -578,17 +569,31 @@ void AggregationTestBase::testReadFromFiles(
   }
 
   auto input = AssertQueryBuilder(builder.planNode()).copyResults(pool());
-  if (input->size() < 2) {
+  if (input->size() <= 0) {
```

Comment:
nit: since input size cannot be negative, this check might be more readable as ==

BTW, when would it be hit?

Reply:
Fixed. Just in case some one adds a test case that passes empty vectors.